src/compiler/utilities.ts

```diff
@@ -3488,8 +3488,8 @@ namespace ts {
     export function escapeString(s: string, quoteChar?: CharacterCodes.doubleQuote | CharacterCodes.singleQuote | CharacterCodes.backtick): string {
         const escapedCharsRegExp =
             quoteChar === CharacterCodes.backtick ? backtickQuoteEscapedCharsRegExp :
-            quoteChar === CharacterCodes.singleQuote ? singleQuoteEscapedCharsRegExp :
-            doubleQuoteEscapedCharsRegExp;
+                quoteChar === CharacterCodes.singleQuote ? singleQuoteEscapedCharsRegExp :
+                    doubleQuoteEscapedCharsRegExp;

```

Comment:
is this prettier at work? *most* of our indentation is on purpose -- we'd prefer to switch to prettier intentionally, not piecemeal.

Reply:
Hmm yes, I can rebase to remove it